bin/docker-image-tool.sh

```diff
@@ -53,7 +53,7 @@ function build {
     # contain a lot of duplicated jars with the main Spark directory. In a proper distribution,
     # the examples directory is cleaned up before generating the distribution tarball, so this
     # issue does not occur.
-    IMG_PATH=resource-managers/kubernetes/docker/src/main/dockerfiles
+    IMG_PATH=resource-managers/kubernetes/docker/src
```

Comment:
Do you still need changes to this file given you have moved the test stuffs out?

Reply:
The dockerfiles and files for building the kerberos/ hadoop docker images are in `src/test`. It still seemed like a logical place to keep them with the `/test` tag, no?

Reply:
I have the same question. It doesn't seem like you're actually using this script for the new test stuff, nor changing any of the existing calls to it, so do you need any of the changes being made here?